tests/e2e/cucumber/sort.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+Then('results are ordered by {string}', (sortOrder: string) => {
+  cy.getByDataTest('sort-dropdown').should('contain', sortOrder);
+});
+
+When('{string} order is clicked', (sortOrder: string) => {
+  cy.getByDataTest('sort-dropdown').click();
+  cy.getByDataTest('dropdown-item').children().contains(sortOrder).click();
```

Comment:
Any reason for the `.children`? I think it is not necessary, and if we remove the `span` in the setup (which I guess we should at some point) it makes the test fail, as it now selects the svg icon